isis/src/base/objs/RadarGroundMap/RadarGroundMap.h

```diff
@@ -117,7 +120,7 @@ namespace Isis {
       virtual bool SetGround(const Latitude &lat, const Longitude &lon);
       virtual bool SetGround(const SurfacePoint &surfacePoint);
       virtual bool GetXY(const SurfacePoint &spoint, double *cudx,
-                         double *cudy);
+                         double *cudy, bool test=false);
```

Comment:
This is going to be accessed polymorphically through the parent CameraGroundMap class which means `test` will default to true instead of false.

https://stackoverflow.com/questions/5273144/c-polymorphism-and-default-argument

I'm not sure what the correct solution is here.